README.md

```diff
@@ -8,7 +8,7 @@ Simulator for Pragyan CodeCharacter 2020
 
 2. Download & Install [Protobuf](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md)
 
-3. Install Clang++ (>=9.0.1)
+3. Install Clang++ (>=9.0.1) (Can install using `sudo apt-get install clang` and also install clang-format using `sudo apt-get install clang-format`)
```

Comment:
Can you make installing clang format as a separate step?

Reply:
Done

Reply:
Move clang-format installation step after cmake-format installation step.

Reply:
> Move clang-format installation step after cmake-format installation step.

Done